Tasks/AzureFileCopy/task.json

```diff
@@ -1,4 +1,4 @@
-﻿{
```

Comment:
most of task.json contain a BOM UTF-FFEF at beginning, it can cause JSON.Parse to throw. I removed them.